Code/GraphMol/Fingerprints/FingerprintGenerator.h

```diff
@@ -259,6 +263,32 @@ class RDKIT_FINGERPRINTS_EXPORT BondInvariantsGenerator
   virtual BondInvariantsGenerator *clone() const = 0;
 };  // namespace RDKit
 
+/*!
+  \brief struct that makes calling the fingerprint generation functions easier
+
+ */
+struct FingerprintFuncArguments {
+  const std::vector<std::uint32_t> *fromAtoms = nullptr;
+  const std::vector<std::uint32_t> *ignoreAtoms = nullptr;
+  int confId = -1;
+  AdditionalOutput *additionalOutput = nullptr;
```

Comment:
We should be clear that FingerprintFuncArguments doesn't own the additionalOutput pointer.  Indeed, I'm finding it hard to track who is responsible for deleting this.

Reply:
That struct doesn't own any of the pointers it stores. I'll add a comment to that effect.